flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/calls/ScalarOperators.scala

```diff
@@ -1288,15 +1290,16 @@ object ScalarOperators {
       nullCheck: Boolean,
       resultType: TypeInformation[_],
       left: GeneratedExpression,
-      right: GeneratedExpression)
+      right: GeneratedExpression,
+      compareNull: Boolean = false)
```

Comment:
Use explicit parameters for all methods in code generation instead of default values.